modules/ROOT/partials/configuration/urlconverter_callback.adoc

```diff
@@ -5,10 +5,10 @@ This option enables you to add your own URL converter logic. This option should
 
 The format of the converter function is: `+URLConverter(url, node, on_save)+`, where:
 
-* url = the URL string to convert,
-* node = the element that contains the URL that is to be converted (this parameter may be set to null if there is no element for the URL), and
-* on_save is always set to true.
-* name = the attribute name that is being set.
+* `+url+`: The URL string to convert.
+* `+node+`: The element that contains the URL that is to be converted (this parameter may be set to null if there is no element for the URL).
+* `+on_save+`: Always set to true.
+* `+name+`: The attribute name that is being set.
```

Comment:
```suggestion
* `+url+` is the URL string to convert.
* `+node+` is the element that contains the URL that is to be converted (this parameter may be set to null if there is no element for the URL).
* `+on_save+` is always set to true.
* `+name+` is the attribute name that is being set.
```
List points should continue on from the sentence above, so making it start new like this didn't make sense so I've changed it to keep the flow.